docs/run/index.md

```diff
@@ -61,3 +69,16 @@ Enters verbose detailed mode with extensive logging.
 `--home`
 
 Specify a custom home directory.
+
+## Start a Blockchain Node in Production
+
+Both `starport serve` and `starport build` compile the source code of the chain in a binary file and install it in `~/go/bin`. By default, the binary name matches the name of the repository with a `d` suffix in the end. For example, if a chain was scaffolded using `starport app github.com/alice/chain`, then the binary will be named `chaind`.
+
+The binary name can be customised in `config.yml`:
```

Comment:
```suggestion
You can customize the binary name in `config.yml`:
```

Reply:
let's use US English spelling https://developers.google.com/style/spelling and active voice to match our other content here (I am loving this getting started guide with friendly inclusive language, good job @fadeev !)